Fst2List.cpp

```diff
@@ -1979,18 +1983,39 @@ int CFstApp::outWordsOfGraph(int depth) {
   return 0;
 }
 
-//
 //
 //
 //
 
-// TODO(jhondoe) Use UnitexGetOpt instead argc, argv
+const char* optstring_Fst2List=":o:Sp:a:t:l:i:mdf:vVhs:r:c:";
+const struct option_TS lopts_Fst2List[]= {
+  {"output",required_argument_TS,NULL,'o'},
+  {"automaton",required_argument_TS,NULL,'a'},
+  {"transducer",required_argument_TS,NULL,'t'},
+  {"limit",required_argument_TS,NULL,'l'},
+  {"stop-graph",required_argument_TS,NULL,'i'},
+  {"print-mode",required_argument_TS,NULL,'p'},
+  {"mode",no_argument_TS,NULL,'m'},
+  {"debug",no_argument_TS,NULL,'d'},
+  {"transducer-mode",required_argument_TS,NULL,'f'},
+  {"separator",required_argument_TS,NULL,'s'},
```

Comment:
This is a bit ambiguous considering the `io_separator`, what about  `lr_delimiters` ?